tbx/project_styleguide/templates/patterns/atoms/sprites/sprites.html

```diff
@@ -42,4 +42,11 @@
     </symbol>
+
+    <symbol id="calendar" viewBox="0 0 54 50" fill="none">
+        <path d="M12.3996 11.4219V15.5552M31.6884 11.4219V15.5552M5.51074 40.3551V19.6885C5.51074 18.5923 5.94622 17.541 6.72136 16.7658C7.49651 15.9907 8.54784 15.5552 9.64406 15.5552H34.444C35.5402 15.5552 36.5915 15.9907 37.3667 16.7658C38.1418 17.541 38.5773 18.5923 38.5773 19.6885V40.3551M5.51074 40.3551C5.51074 41.4513 5.94622 42.5026 6.72136 43.2778C7.49651 44.0529 8.54784 44.4884 9.64406 44.4884H34.444C35.5402 44.4884 36.5915 44.0529 37.3667 43.2778C38.1418 42.5026 38.5773 41.4513 38.5773 40.3551M5.51074 40.3551V26.5774C5.51074 25.4811 5.94622 24.4298 6.72136 23.6547C7.49651 22.8795 8.54784 22.4441 9.64406 22.4441H34.444C35.5402 22.4441 36.5915 22.8795 37.3667 23.6547C38.1418 24.4298 38.5773 25.4811 38.5773 26.5774V40.3551" stroke="var(--color--accent-two)" stroke-width="2"/>
```

Comment:
I didn't know that using CSS variables directly here would work. I think it might be slightly clearer to use `stroke="currentColor"` then set the correct colour in the css - although you'd have to either split the svg into two, or add classes to your paths for that to work.

Reply:
Using classes don't seem to work, something to do with how `use` inherits styles maybe?

Reply:
Discussed on slack and agreed to leave this as it is.